lambdas/archive-vcs/src/main/java/README.md

```diff
@@ -0,0 +1,12 @@
+# ArchiveVcsHandler
+
+To run this lambda, create an appropriate payload including userIds with kbv vcs in the the appropriate environment.
```

Comment:
Might be useful to say what this actually _does_, but perhaps just link to the ticket to avoid duplication.

We should also be a little bit careful about what information we put here, since it's a public repo :)